### PR TITLE
[CityTab] Increase FlatList’s window size to avoid blank views.

### PR DESCRIPTION
**Pull request by alloy**
Fixes https://artsyproduct.atlassian.net/browse/LD-362

#skip_new_tests

### CODE REVIEW

File: src/lib/Scenes/City/Components/CityTab.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export class CityTab extends React.Component<Props> {
         keyExtractor={item => item.node.id}
         renderItem={({ item }) => this.renderItem(item)}
         scrollEnabled={false}
+        windowSize={50}
```
Comment (alloy) [inline src/lib/Scenes/City/Components/CityTab.tsx]: An hour of faffing with Xcode and all I got was this 1 line change

Comment (orta) [reply src/lib/Scenes/City/Components/CityTab.tsx]: bah

Comment (alloy) [reply src/lib/Scenes/City/Components/CityTab.tsx]: To be fair, that was mostly just me catching up with the status quo in general and specifically on my iMac

Comment (orta) [reply src/lib/Scenes/City/Components/CityTab.tsx]: hab

Comment (l2succes) [reply src/lib/Scenes/City/Components/CityTab.tsx]: wow, one liner fixes 😩

**Pull request merged**